app/src/main/AndroidManifest.xml

```diff
@@ -45,8 +45,7 @@
         <activity
             android:name=".ViewActivity"
             android:configChanges="orientation|screenLayout|screenSize|smallestScreenSize|keyboardHidden"
-            android:exported="true"
-            android:launchMode="singleTop">
```

Comment:
Guess we want singleTask then...

Reply:
Nope, because singleTask behaviour is similar to singleTop (recycle activities if one already exist in the stack), we really just want standard to make sure every new intent uses a new instance (this is also made to make sure we don't use non-secure activities for secure viewing)